src/main/drivers/pwm_output_stm32f7xx.c

```diff
@@ -236,9 +236,13 @@ void pwmDigitalMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t
 
     /* PWM1 Mode configuration: Channel1 */
     TIM_OCInitStructure.OCMode = TIM_OCMODE_PWM1;
-    TIM_OCInitStructure.OCPolarity = TIM_OCPOLARITY_HIGH;
-    TIM_OCInitStructure.OCIdleState = TIM_OCIDLESTATE_RESET;
-    TIM_OCInitStructure.OCNIdleState = TIM_OCNIDLESTATE_RESET;
+    if (output & TIMER_OUTPUT_N_CHANNEL) {
```

Comment:
Need someone to review these lines, I don't have real F7 hardware to test on.
Maybe @sambas could help?